include/volt.h

```diff
@@ -9,9 +9,39 @@
 #include <Arduino.h>
 #include <farmSensor.h>
 
+class volt : public farmSensor {
+private:
+    int m_RH; // voltage divider, higher resistor value [ohms]
+    int m_RL; // voltage divider, lower resistor value [ohms]
+protected:
+    float readRaw() override; 
+public:
+    static const int adc_res = 4095; // ESP32 ADC resolution is 12 bits
+    volt(int pin, sensorType type, String name, String unit, char shortcut, int RL, int RH);
+    void initialize() override;
+};
 
-class volt
+/* constructor */
+volt::volt(int pin, sensorType type, String name, String unit, char shortcut, int RL, int RH): 
+            farmSensor(pin, type, name, unit, shortcut), m_RH(RH), m_RL(RL) {}
```

Comment:
Nice!
I like how you utilized the inherited class constructor here. I should do this in `counter.h`
Good stuff :)